apps/project/tests/test_filters.py

```diff
@@ -0,0 +1,91 @@
+from utils.graphene.tests import GraphQLTestCase
+
+from project.factories import ProjectFactory
+from organization.factories import OrganizationFactory
+from analysis_framework.factories import AnalysisFrameworkFactory
+
+from project.filter_set import ProjectGqlFilterSet
+from project.models import Project
+
+ACTIVE = Project.Status.ACTIVE
+INACTIVE = Project.Status.INACTIVE
+
+
+class TestProjectFilter(GraphQLTestCase):
+    def setUp(self) -> None:
+        self.filter_class = ProjectGqlFilterSet
+
+    def test_organization_filter(self):
+        org1 = OrganizationFactory.create()
+        org2 = OrganizationFactory.create()
+        org3 = OrganizationFactory.create()
+        p1 = ProjectFactory.create()
+        p1.organizations.set([org2, org1])
+        p2 = ProjectFactory.create()
+        p2.organizations.set([org2, org3])
+        p3 = ProjectFactory.create()
+        p3.organizations.add(org1)
```

Comment:
Looks like we can use create_batch here as well. Can you check?
```suggestion
        org1, org2, org3 = OrganizationFactory.create_batch(3)
        p1, p2, p3 = ProjectFactory.create_batch(3)
        p1.organizations.set([org2, org1])
        p2.organizations.set([org2, org3])
        p3.organizations.add(org1)
```